src/_includes/head.html

```diff
@@ -21,7 +21,11 @@
 
   <!-- tilt.dev typekit project -->
   <link rel="stylesheet" href="https://use.typekit.net/yii5fqs.css">
-  <link href="https://fonts.googleapis.com/css2?family=Fira+Code&display=swap" rel="stylesheet">
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@300&display=swap" rel="stylesheet">
```

Comment:
I'm specifying the exact weight of Fira Code we want to use. Also, I guess Google Fonts updated the boilerplate they ask you to copy. Supposedly this results in faster perceived loading time?

Reply:
It might be good to include a heavier weight as well - I noticed recently that if you have `<strong><code>foo</code></strong>` that it won't get bolded:

<img width="803" alt="screenshot of monospace font not being bolded when wrapped in a strong tag" src="https://user-images.githubusercontent.com/841263/136567642-9f3e54b9-fcf2-46e9-85ca-64a79ecf89b2.png">

(notice the parameter names are not bold)

Reply:
Thanks, @milas! I included a bold version of Fira Code.